src/manager.cpp

```diff
@@ -39,6 +39,11 @@ Manager::Manager(
 
         // set async timer to detect if VPD collection is done.
         SetTimerToDetectVpdCollectionStatus();
+
+        const nlohmann::json& l_sysCfgJsonObj = m_worker->getSysCfgJsonObj();
```

Comment:
Don't need this variable as it is not being used at multiple places. Please get this in place.

Reply:
Updated